src/components/forms/Register.js

```diff
@@ -99,59 +103,111 @@ const Register = () => {
   }
   return (
     <>
+    <h2 style={styles.heading}>Register for Student ArtCo!</h2>
+     <Box display='flex' style={styles.textfieldbox}>
       <form onSubmit={onSubmit}>
-        <input
+        <TextField
+          variant='outlined'
+          label='Email'
+          style={styles.textfield}
+          size='small'
+          fullWidth={false}
           type='text'
           name='email'
           value={email}
-          placeholder='email'
           onChange={e => setEmail(e.target.value)}
-          required
         />
-        <input
+        <TextField
+          variant='outlined'
+          label='Password'
+          style={styles.textfield}
+          size='small'
+          fullWidth={false}
           type='text'
```

Comment:
Should this be type password is there something with material that is hiding the password as you type?

Reply:
I agree. This should be a type `'password'` so that it is hidden as the user types it out. Even though this is a registration form the standard practice is to use the type of `'password'` on these fields and hide it as someone is setting their password for their account.

Reply:
Yes you're right password and email would be the correct type attributes. Fixing now, thanks!